src/modules/Laboratorio/Visualizar/components/editar.js

```diff
@@ -0,0 +1,64 @@
+import React, {useState, useEffect} from 'react';
+import Menu from '../../../../utils/components/Menu';
+import Title from '../../../../utils/components/Title';
+import Toolbar from '../../../../utils/components/Toolbar';
+import Formulario from './Formulario';
+import {Redirect} from 'react-router-dom';
+import api from '../../../../services/api';
+
+
+const Editar =  (props) => {
+  const [shouldRedirect,setShouldRedirect] = useState(false);
+  const [category, getCategory] = useState([]);
+  const [location, getLocation] = useState([]);
```

Comment:
Normalmente a comunidade usa o padrão `[something, setSomething]` com o hook `useState()`. Acho que seria uma boa utilizar aqui também.

```suggestion
  const [category, setCategory] = useState([]);
  const [location, setLocation] = useState([]);
```